src/components/EolPage.js

```diff
@@ -16,6 +16,7 @@ const releaseNotesQuery = graphql`
       }
     ) {
       nodes {
+        fileAbsolutePath
         frontmatter {
           releaseDate
           subject
```

Comment:
`super nit`: we can get rid of subject since its not being used 🤷

Reply:
ah good catch. will remove